packages/manager/modules/billing-components/src/components/services-actions/services-actions.controller.js

```diff
@@ -150,4 +152,12 @@ export default class ServicesActionsCtrl {
       this.atInternet.trackClick({ name, type: 'action' });
     }
   }
+
+  handleClickResiliate() {
+    this.trackAction('go-to-resiliate');
```

Comment:
Why do you add tracking here ? Is it a requirement ?

Reply:
It's already applied on precedent behavior. It's to avoid a regression